doc/developer-guides/hld/hv-hypercall.rst

```diff
@@ -3,39 +3,40 @@
 Hypercall / HSM Upcall
 ######################
 
-The hypercall/upcall is used to request services between the Guest VM and the hypervisor.
-The hypercall is from the Guest VM to hypervisor, and the upcall is from the hypervisor to the Guest VM.
-The hypervisor currently supports hypercall APIs for VM management, I/O request
-distribution, interrupt injection, PCI assignment, guest memory mapping,
-power management, and secure world switch.
+The hypercall/upcall is used to request services between the guest VM and the
+hypervisor. The hypercall is from the guest VM to the hypervisor, and the upcall
+is from the hypervisor to the guest VM. The hypervisor currently supports
```

Comment:
Let's avoid using "currently" or implying that a future release will do something different:
```suggestion
is from the hypervisor to the guest VM. The hypervisor supports
```